jupyterhub/values.yaml

```diff
@@ -63,10 +63,10 @@ hub:
       accessModes:
         - ReadWriteOnce
       storage: 1Gi
-      subPath:
-      storageClassName:
-    url:
-    password:
+      subPath: ""
+      storageClassName: ""
```

Comment:
If it's set to empty rather than unset, will the resulting field be present? I think many fields in Kubernetes have adifferent effect when it's set to empty string - particularly, the defaults might not apply. 

For example, an empty storageClassName has different behavior from having it be not present (see [doc](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reserving-a-persistentvolume)). 

So my instinct is to allow anything that can be a non-container type (so strings, numbers, bools) be nullable.

Reply:
I agree, I think null for unset is often the right thing to do.

Reply:
Ah yes storageClassName was one of those fields where we differentiate between unset and empty string!

```yaml
  {{- if typeIs "string" .Values.hub.db.pvc.storageClassName }}
  storageClassName: {{ .Values.hub.db.pvc.storageClassName | quote }}
  {{- end }}
```

I agree. Resolved by 1153aa35.